test/karma/firefox.js

```diff
@@ -0,0 +1,8 @@
+var common = require('./common');
+
+module.exports = function(config) {
+  config.set(common({
+    plugins: ['karma-firefox-launcher'],
+    browsers: ['Firefox']
+  }));
+};
```

Comment:
Can we just combine all of these individual browser modules into a single file? They're only eight lines apiece.